docs/overview/Glossary.md

```diff
@@ -2,226 +2,209 @@
 
-### election guardian
+### election key pair
 
-One of a number of independent, trustworthy individuals who, while serving as an _election guardian_ in an [end-to-end verifiable election](#end-to-end-verifiable-election), participate in a [key ceremony](#key-ceremony) to create a [joint public election key](#joint-key) and perform a [tally](#tally)
+A [key pair](#key-pair) owned by a specific [guardian](#guardian) for joint encryption in combination with the other guardians to secure the election.
 
-### election manifest (see [manifest](#manifest))
+###### _Other terms:_ election keys, guardian key pair, guardian keys
 
 ### election partial key backup
 
-A point on a secret polynomial and commitments to verify this point for a designated guardian.
+A point on a secret [election polynomial](#election-polynomial) and commitments to verify this point for a designated guardian.
 
-### election polynomial
+###### _Other terms:_ backup
 
-The _election polynomial_ is the mathematical expression that each Guardian defines to solve for his or her private key. A different point associated with the polynomial is shared with each of the other guardians so that the guardians can come together to derive the polynomial function and solve for the private key.
+### election partial key verification
 
-### encrypt
+A [guardian's](#guardian) verification that a received [election partial key backup](#election-partial-key-backup) is valid to ensure decryption can occur with missing guardians in the tally ceremony.
 
-### encrypted ballot metadata
+###### _Other terms:_ backup verification
 
-A specific subset of election metadata
+### election polynomial
 
-### encrypted tally
+The election polynomial is the mathematical expression that each [guardian](#guardian) uses for encryption. Each guardian has a polynomial where the first coefficient is used to generate their [election key pair](#election-key-pair). A different point associated with the polynomial is shared with each of the other guardians in the [key ceremony](#key-ceremony) so that the guardians can come together and decrypt in the [tally ceremony](#tally-ceremony).
 
-The homomorphically-combined and encrypted representation of all selections made for each option on every contest in the election.
+### election record
 
-### encrypting a ballot
+A verifiable record of the public artifacts or files of the election. This includes items like the manifest and the encrypted ballots so an individual or third party can verify the election end-to-end.
 
-As outlined in the ElectionGuard Specification
+###### _Other terms:_ record, election artifacts _(deprecated)_
 
-```mermaid
-    graph LR
-    a1(plaintext ballot) -->|encrypt| a2(cipertext ballot)
-```
+### ElectionGuard Core
```

Comment:
Took new approach. 

Defined what an ElectionGuard Core, SDK, and Specification mean. These have been a kind of general concepts that we discuss but were not written. Used capital "Core" to avoid the overlap with the actual word "core". 

Also stuck with Path over stack. This does make more sense.